### PR TITLE
Fixed embeds and some changes

### PR DESCRIPTION
**Pull request by sayand0122**
Fixed up the remaining embeds and added environment variables.

### CODE REVIEW

**Comment by Vyvy-vi** [comment]
closes #67

File: bot/main.py [hunk 1]
```diff
@@ -1,37 +1,37 @@
 # TODO - transfer, casino, etc commands
 
 
+import aiohttp
+import time
+import random
+import wikipedia
```
Comment (Vyvy-vi) [inline bot/main.py:7]: looks better formatted 👍

File: bot/main.py [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
 
+load_dotenv()
 
 # temp->
-from inputs import responses, fortunes, quo, nerd, tech, rost, bk, cmp, blurt, cf, jk, cfe, chill, cl, ur
 
 # modules for wiki and wolfram queries
 # import wolframalpha
-import wikipedia
 # import requests
```
Comment (Vyvy-vi) [inline bot/main.py:22]: maybe shift the comments to their right places

File: bot/main.py [hunk 3]
```diff
@@ -101,7 +102,9 @@ async def on_guild_join(guild):
         if channel.permissions_for(guild.me).send_messages:
             if guild.id not in db.list_collection_names():
                 col = db[str(guild.id)]
-                col.insert_one({'server_name': guild.name, 'server_id': guild.id})
+                col.insert_one(
+                    {'server_name': guild.name, 'server_id': guild.id})
+            icon_url = 'https://cdn.discordapp.com/attachments/582605227081990233/627388598181953548/unknown.png'
```
Comment (Vyvy-vi) [inline bot/main.py:105]: I added the variable as using this might be more neat

**Review by sayand0122 — COMMENTED** [review]
👍🏻

**Review by Vyvy-vi — APPROVED** [review]

**Comment by Vyvy-vi** [comment]
LGTM!

**Comment by sayand0122** [comment]
Neat and Nice!!!

**Comment by Vyvy-vi** [comment]
@all-contributors add @sayand0122 for code and design

**Pull request merged**